### ISSUE TITLE
ISIN standard

### ISSUE THREAD

**Issue by 2martin2** [SimplyWallSt/Company-Analysis-Model#27]
Hi, is there any reason why you are not using the ISIN standard?

**Comment by viktor-ferenczi** [comment]
Searched by company name because I wasn't aware of the NYSE:T standard
naming.

I had to look it up somewhere else to find AT&T on your service, which is I
think suboptimal.

Maybe 100% of americans knew this already, but I did not. (I'm from the EU).

You may want to special case AT&T and any other company names with special
characters in it (H&M comes to mind), so it still works as the user would
expect and the company is at least found by the search box.

**Comment by swvanderlaan** [comment]
I was wondering the same. Seems like using the `ISIN` standard will circumvent many of the problems with using ticker names in combination with stock exchange symbols. For instance `ENXTAM` is one way of noting `EAM`, _i.e._ the Amsterdam Stock Exchange. `ISIN` is a unique number.

**Comment by maartenvanpottelberghe** [comment]
yeah, i totally second this. using ISIN would make life so much easier for most of us. one unique identifier to rule them all....
now cannot use the feature i am interested in most. will make me think twice before i spend 80 bucks for an account,...
